--- plugins/misc/xcscope.el	2012-02-21 17:25:46.641046893 -0800
+++ my-xcscope.el	2012-02-21 17:24:07.473601528 -0800
@@ -1014,6 +1014,15 @@
   (define-key cscope:map "\C-csT" 'cscope-tell-user-about-directory)
   (define-key cscope:map "\C-csD" 'cscope-dired-directory))
 
+;; add another bindling, using \C-; as prefix key
+(define-prefix-command 'my-cscope-map)
+(global-set-key (kbd "C-;") 'my-cscope-map)
+(define-key my-cscope-map (kbd "s") 'cscope-find-this-symbol)
+(define-key my-cscope-map (kbd "d") 'cscope-find-global-definition)
+(define-key my-cscope-map (kbd "c") 'cscope-find-functions-calling-this-function)
+(define-key my-cscope-map (kbd "C") 'cscope-find-called-functions)
+(define-key my-cscope-map (kbd "e") 'cscope-find-egrep-pattern)
+
 (easy-menu-define cscope:menu
 		  (list cscope:map cscope-list-entry-keymap)
 		  "cscope menu"
