--- plugins/misc/xcscope.el	2002-01-09 10:50:30.000000000 -0800
+++ my-xcscope.el	2012-02-21 11:56:33.967650153 -0800
@@ -995,14 +995,22 @@
   "The cscope keymap.")
 (if cscope:map
     nil
-  (setq cscope:map (make-sparse-keymap))
-  (define-key cscope:map "\C-css" 'cscope-find-this-symbol)
-  (define-key cscope:map "\C-csd" 'cscope-find-global-definition)
+  ;;(setq cscope:map (make-sparse-keymap))
+  ;; overwrite key bindings, use C-' as prefix key, since it's easier to press :)
+  (define-prefix-command 'cscope:map)
+  (global-set-key (kbd "C-'") 'cscope:map)
+  ;;(define-key cscope:map "\C-css" 'cscope-find-this-symbol)
+  (define-key cscope:map (kbd "s") 'cscope-find-this-symbol)
+  ;;(define-key cscope:map "\C-csd" 'cscope-find-global-definition)
+  (define-key cscope:map (kbd "d") 'cscope-find-global-definition)
   (define-key cscope:map "\C-csg" 'cscope-find-global-definition)
-  (define-key cscope:map "\C-csc" 'cscope-find-functions-calling-this-function)
-  (define-key cscope:map "\C-csC" 'cscope-find-called-functions)
+  ;;(define-key cscope:map "\C-csc" 'cscope-find-functions-calling-this-function)
+  (define-key cscope:map (kbd "c") 'cscope-find-functions-calling-this-function)
+  ;;(define-key cscope:map "\C-csC" 'cscope-find-called-functions)
+  (define-key cscope:map (kbd "C") 'cscope-find-called-functions)
   (define-key cscope:map "\C-cst" 'cscope-find-this-text-string)
-  (define-key cscope:map "\C-cse" 'cscope-find-egrep-pattern)
+  ;;(define-key cscope:map "\C-cse" 'cscope-find-egrep-pattern)
+  (define-key cscope:map (kbd "e") 'cscope-find-egrep-pattern)
   (define-key cscope:map "\C-csf" 'cscope-find-this-file)
   (define-key cscope:map "\C-csi" 'cscope-find-files-including-file)
   ;;
