--- zjl-hl.el	2010-05-30 00:40:12.000000000 -0700
+++ plugins/zjl-hl/zjl-hl.el	2012-03-14 17:53:51.079696359 -0700
@@ -109,7 +109,7 @@
 (defface zjl-hl-function-call-face
   '((((class color)
       (background dark))
-     (:foreground "#e566d7" :weight normal))
+     (:foreground "forestgreen" :bold t))
     (((class color)
       (background light))
      (:foreground "#008000" :bold t))
@@ -123,7 +123,7 @@
 (defface zjl-hl-local-variable-reference-face
   '((((class color)
       (background dark))
-     (:foreground "LightGoldenrod"))
+     (:foreground "darkorange"))
     (((class color)
       (background light))
      (:foreground "#008080"))
@@ -137,7 +137,7 @@
 (defface zjl-hl-parameters-reference-face
   '((((class color)
       (background dark))
-     (:foreground "LightGoldenrod" :weight bold))
+     (:foreground "tomato"))
     (((class color)
       (background light))
      (:foreground "#008080" :bold t))
